test/test_braid.py

```diff
@@ -1,6 +1,6 @@
 import sys
 import os
-
+sys.path.append("..")
```

Comment:
This is not the correct way of importing local library under test, we should add its path to PYTHONPATH environment variable.